presto-hive/src/test/java/com/facebook/presto/hive/s3/MockAmazonS3.java

```diff
@@ -81,6 +90,19 @@ public PutObjectResult putObject(String bucketName, String key, String content)
         return new PutObjectResult();
     }
 
+    @Override
+    public ListObjectsV2Result listObjectsV2(String bucketName, String prefix)
```

Comment:
Why do we need `listObjectsV2`? It seems that both `ObjectListing` and `ListObjectsV2Result` contain the same `S3ObjectSummary` object which has storage class.

Reply:
Will switch to use listObjects.